en/computer_vision/collision_prevention.md

```diff
@@ -178,6 +179,7 @@ The system may work well at significantly higher speeds and lower frequency dist
 :::
 
 The tested companion software is the _local_planner_ from the [PX4/PX4-Avoidance](https://github.com/PX4/PX4-Avoidance) repo.
```

Comment:
@hamishwillee should we even mention Avoidance? As it's no longer maintained, it might not be sensible to mention it.

Reply:
Nope. Fixed.